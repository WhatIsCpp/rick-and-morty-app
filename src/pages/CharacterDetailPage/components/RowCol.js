import React, { Component } from 'react';
import { Row, Col } from 'antd';

export default class RowCol extends Component {
  render() {
      const {label, value} = this.props;
    return (
      <Row>
        <Col span={2} className="label">
          {label}
        </Col>
        <Col span={20} className="value">
          {value}
        </Col>
      </Row>
    );
  }
}
