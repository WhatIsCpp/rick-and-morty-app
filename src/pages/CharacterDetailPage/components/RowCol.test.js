import React from 'react';
import ReactDOM from 'react-dom';
import RowCol from './RowCol';
import renderer from 'react-test-renderer';
it('RowCol tests passed successfully', () => {
  const div = document.createElement('div');
  ReactDOM.render(<RowCol />, div);
});

it('Snapshot Created', () => {
  let props = { label: 'Name', value: 'Furkan' };
  const component = renderer.create(<RowCol {...props} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
