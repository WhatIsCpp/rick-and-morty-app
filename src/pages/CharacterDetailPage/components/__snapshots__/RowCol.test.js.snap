// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RowCol tests passed successfully 1`] = `
<div
  className="ant-row"
>
  <div
    className="ant-col-2 label"
  >
    Name
  </div>
  <div
    className="ant-col-20 value"
  >
    Furkan
  </div>
</div>
`;

exports[`RowCol tests passed successfully 2`] = `
<div
  className="ant-row"
>
  <div
    className="ant-col-2 label"
  >
    Name
  </div>
  <div
    className="ant-col-20 value"
  >
    Furkan
  </div>
</div>
`;

exports[`Snapshot Created 1`] = `
<div
  className="ant-row"
>
  <div
    className="ant-col-2 label"
  >
    Name
  </div>
  <div
    className="ant-col-20 value"
  >
    Furkan
  </div>
</div>
`;
